### PR TITLE
[IMP] General: Multi-Company Limitations

### PR DESCRIPTION
**Pull request by jero-odoo**
Adding additional details to the Multi-Company doc regarding limitations.

Docs task: ﻿[﻿https://www.odoo.com/mail/view?model=project.task&res_id=4070910&access_token=0ee2f233-2bb1-4772-9bd2-0277f56aeaf8](https://www.odoo.com/mail/view?model=project.task&res_id=4070910&access_token=0ee2f233-2bb1-4772-9bd2-0277f56aeaf8)

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hey @pabr-odoo this is ready for peer review. (Just a small addition). Thanks!

**Review by pabr-odoo — APPROVED** [review]
Hey @jero-odoo just finished reviewing, great addition! It all looks good to me, so feel free to pass it on for final review.

**Comment by jero-odoo** [comment]
Thanks Parish!
@ksc-odoo this is ready for final review! Thanks!

File: content/applications/general/multi_company.rst [hunk 1]
```diff
@@ -54,6 +54,8 @@ except under the following circumstances:
    When he creates a new sales team, the :guilabel:`Company` field automatically defaults to `My
```
Comment (ksc-odoo) [inline content/applications/general/multi_company.rst:54]:
```suggestion
   When a new sales team is created, the :guilabel:`Company` field automatically defaults to `My
```

File: content/applications/general/multi_company.rst [hunk 2]
```diff
@@ -54,6 +54,8 @@ except under the following circumstances:
...
    Company (Chicago)`.
 
+.. _general/sharing-data:
+
 Sharing data
```
Comment (ksc-odoo) [inline content/applications/general/multi_company.rst:59]:
```suggestion
Share data
```

**Review by ksc-odoo — APPROVED** [review]
Hey, @jero-odoo -- just finished my Final Review of this PR. Nicely done. Just a couple of minor things to consider during your revisions, although, they're mainly for the other portion of the doc that you didn't adjust in this PR. Other than that, when you think this one is ready to go, feel free to move it to the next stage. Thanks!

**Comment by jero-odoo** [comment]
Hey @StraubCreative this is ready for tech review. Thanks!

**Review by samueljlieber — APPROVED** [review]
Looks good @jero-odoo, thank you!
..
@robodoo r+

**Pull request closed without merging**